service/src/test/java/uk/gov/hmcts/reform/fpl/service/ValidateEmailServiceTest.java

```diff
@@ -13,10 +12,11 @@
 import java.util.Optional;
 import java.util.stream.Stream;
 
+import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThat;
 
 @ExtendWith(SpringExtension.class)
-@ContextConfiguration(classes = { ValidateEmailService.class })
+@ContextConfiguration(classes = {ValidateEmailService.class})
```

Comment:
Don't think we need to have the spring extension + this, there aren't any dependencies in the service

Reply:
done